acm/overlays/moc/infra/placements/argocd-managed-clusters.yaml

```diff
@@ -4,7 +4,9 @@ metadata:
   name: argocd-managed-clusters
   namespace: argocd
 spec:
-  clusterSets:
-    - argocd-managed
-    - octo-ushift-dev
-    - osc
+  predicates:
+    - requiredClusterSelector:
+        labelSelector:
+          matchExpressions:
+            - key: cloud
+              opreator: Exists
```

Comment:
`operator` maybe?

Reply:
Did you try with an empty labelSelector as Scott suggested ?

Reply:
@itdove so something like: 

`labelSelector: {}` ?

Reply:
yes, let me quickly try in my env.

Reply:
yes it works, I set the label selector as above and my current cluster kept being selected. I also import an new cluster and set the clusterset with no additional label and it get selected.

Reply:
okay thanks @itdove !!

Reply:
Thanks for taking the time to test it out @itdove I've updated the above.